mithril-common/src/cardano_transactions_preloader.rs

```diff
@@ -32,18 +60,28 @@ impl CardanoTransactionsPreloader {
         security_parameter: BlockNumber,
         chain_observer: Arc<dyn ChainObserver>,
         logger: Logger,
+        activation_state: Arc<dyn CardanoTransactionsPreloaderActivationState>,
     ) -> Self {
         Self {
             signed_entity_type_lock,
             importer,
             security_parameter,
             chain_observer,
             logger,
+            activation_state,
         }
     }
 
     /// Preload the Cardano Transactions by running the importer up to the current chain block number.
     pub async fn preload(&self) -> StdResult<()> {
+        if !self.is_activated().await? {
+            info!(
+                self.logger,
+                "🔥 Preload Cardano Transactions - Not running, conditions not met"
```

Comment:
Can we replace all the :fire: with a :warning: or :information_source: in the preloader? 
This will help avoid confusion IMHO

Also this log should be removed or kept as a trace.

Reply:
The idea with adding emoji is to make the service stand up in the log. So if we use one we should choose it carefully so it's not reused by another service.

Also I think that a info or debug is the right level, trace seems a bit too low since this log won't be spammed and have business related.

Beyond that I rethought about our log usage and ... we probably are using them wrongly.

The main culprit is `slog_scope`, we should only use it for logs not scoped in a service and all services should have a child logger (created using [`Logger::new(&self, values)`](https://docs.rs/slog/latest/slog/struct.Logger.html#method.new)) with a `name` property that identify them (we should also set some property on the root logger).

Basically we should do this:
* At the global logger creation:
```rust
 Logger::root(Arc::new(drain), slog::o!("name" => "aggregator"));
``` 
* When creating a service
```rust
let verifier = Arc::new(MithrilCertificateVerifier::new(
    self.get_logger()?.new(o!("name" => "certificate_verifier")),,
    self.get_certificate_repository().await?,
));
``` 

Doing so our logs instead of looking like this:
```json
{"msg":"EpochService::update_protocol_parameters","v":0,"name":"slog-rs","level":20} 
``` 
Our logs could look like this:
```json
{"msg":"update_protocol_parameters","v":0,"name":"EpochService","level":20} 
```

Reply:
Even better, the [Logger::new(&self, values)](https://docs.rs/slog/latest/slog/struct.Logger.html#method.new) method could be called by the service constructor so nothing change in the builder and the service decide itself the name that it want to use.

Reply:
We are on the same page, the :fire: is simply not a very judicious choice in that case: it could be used in case of a crash or something going wrong in the node, which is not the case. :warning: or :information_source: where just suggestions.

I keep thinking that this specific log should not be displayed in info, but debug is probably a good match instead of trace.

And yes, we can probably make some refactoring on the logs as you propose.

Reply:
For the emoji I would prefer something that can be related to "preloading", like: ["⟲"](https://symbl.cc/en/27F2/), ["⟳"](https://symbl.cc/en/27F3/), or ["⤾"](https://symbl.cc/en/293E/).

Reply:
Done ✅